{"name": "Тест на невозможность создать двух курьеров с одинаковым логином", "status": "passed", "start": 1730879863928, "stop": 1730879864105, "uuid": "31fff8e8-0333-493a-b548-d90fa83dca3a", "historyId": "91650e0927ed7aafd9ef9ed4a7f8915a", "testCaseId": "91650e0927ed7aafd9ef9ed4a7f8915a", "fullName": "tests.couriers.test_create_couriers.TestCreateCourier#test_create_two_couriers_with_same_login_error", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_create_couriers"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "R-0156"}, {"name": "thread", "value": "31326-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_create_couriers"}]}